### DISCUSSION TITLE
Failed to update DBeaver from 22.0.5.202205220912 to 23.3.3

### DISCUSSION THREAD

**Discussion by vschenrun** [dbeaver/dbeaver#22660] [Q&A] [answered]
### Description

社区版本(22.0.5.202205220912)升级到23.3.3无效,升级安装了23.3.3的包,重新启动软件显示的版本还是22.0.5.202205220912,手动检测更新继续给出23.3.3 最新版本,麻烦帮忙测试一下是否存在版本升级的问题!

版本信息如下:
新的DBeaver版本可用
当前版本: 22.0.5.202205220912
新版本:
233.3
(22.01.2024)

### DBeaver Version

社区版本 22.0.5.202205220912

### Operating System

windows11

### Steps to reproduce

社区版本(22.0.5.202205220912)在线升级到23.3.3无效,重启后还是22.0.5.202205220912版本

**Comment by E1izabeth** [comment]
Could you translate your issue into English?

How did you install DBeaver (with exe installer, with Microsoft Store, or use portable version - zip)?

**Comment by vschenrun** [comment]
The community version (22.0.5.202205220912) upgrade to 23.3.3 does not work, the upgrade installed 23.3.3 package, restart the software shows the version of 22.0.5.202205220912, manual detection updates continue to give 23.3.3 The latest version, please help to test whether there is a version upgrade problem!

Version information is as follows:
New DBeaver version available
Current version: 22.0.5.202205220912
New version:
233.3
(22.01.2024)

DBeaver Version
Community version 22.0.5.202205220912

Operating System
windows11

Database and driver
No response

Steps to reproduce
Community version (22.0.5.202205220912) online upgrade to 23.3.3 does not work, after the restart is still 22.0.5.202205220912 version

Additional context
No response

**Comment by vschenrun** [comment]
https://dbeaver.io/download/ 
![image](https://github.com/dbeaver/dbeaver/assets/24557629/a399903f-a2aa-4791-8cbe-ea8f681e83ed)
![image](https://github.com/dbeaver/dbeaver/assets/24557629/c355c3a9-d8df-435e-96f7-4532807b5e7f)
![image](https://github.com/dbeaver/dbeaver/assets/24557629/fbbbe087-2d4a-4829-b410-4fd99b0bd7e3)
![image](https://github.com/dbeaver/dbeaver/assets/24557629/725681e9-b936-4390-93b1-b849c7b89287)

**Comment by vschenrun** [comment]
> Could you translate your issue into English?
> 
> How did you install DBeaver (with exe installer, with Microsoft Store, or use portable version - zip)?

Upgrade steps with the help of dbeaver. Detection update -&gt; Upgrade -&gt; Install the app - Finished.
Restarting the app still shows version 22.0.5

**Comment by E1izabeth** [comment, marked as answer]
When you run the installer, there is a choice of the installation way and place
![image](https://github.com/dbeaver/dbeaver/assets/28875055/21233110-2618-49ed-ab13-7e7dda596d49)

Is it possible that you chose an option other than the one selected during the initial installation? I suspect that there are 2 applications installed on your computer now - one installed for all users and another one installed for current user.

Could you check? To check it, you can try changing this switcher in the update wizard.

**Comment by vschenrun** [comment]
> When you run the installer, there is a choice of the installation way and place ![image](https://private-user-images.githubusercontent.com/28875055/301458961-21233110-2618-49ed-ab13-7e7dda596d49.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY3ODY1NDgsIm5iZiI6MTcwNjc4NjI0OCwicGF0aCI6Ii8yODg3NTA1NS8zMDE0NTg5NjEtMjEyMzMxMTAtMjYxOC00OWVkLWFiMTMtN2U3ZGRhNTk2ZDQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjAxVDExMTcyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczZTIxZDgyMjFiMDYyOTdiYTA4Zjk1YmRkOGI0ZjY1Y2Y1N2ZhMGQxMGI1ZTA5Mjg0YjNlOTE4MzczNmZlMDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5e14qvUEb0z1Ux6sd5bYFnb3ZbKZTuj_U6rwGqbjjDs)
> 
> Is it possible that you chose an option other than the one selected during the initial installation? I suspect that there are 2 applications installed on your computer now - one installed for all users and another one installed for current user.
> 
> Could you check? To check it, you can try changing this switcher in the update wizard.
I uninstall and reinstall, now is the latest version, thank you!
